### PR TITLE
feat: Bulk add cases to suite

### PR DESCRIPTION
**Pull request by Goblenus**
It solves my problem then I need to add 10k+ cases to 1 suite. Now it takes hours.

### CODE REVIEW

File: junitparser/junitparser.py [hunk 1]
```diff
@@ -470,6 +470,12 @@ def add_testcase(self, testcase):
         self.append(testcase)
         self.update_statistics()
 
+    def add_testcases(self, testcases):
+        """Adds a testcases to the suite."""
+        for testcase in testcases:
```
Comment (EnricoMi) [inline junitparser/junitparser.py:475]: The loop could be made redundant if there would be a `def extend` that calls into `self._elem.extend`, which is the equivalent of `append` for a list.

**Review by EnricoMi — COMMENTED** [review]
minor comments

File: junitparser/junitparser.py (hunk 1 above)
Comment (Goblenus) [reply junitparser/junitparser.py:475]: It cab be done only that way
``` python
def extend(self, sub_elems):
        self._elem.extend((sub_elem._elem for sub_elem in sub_elems))
```
Is it ok?

File: junitparser/junitparser.py [hunk 2]
```diff
@@ -470,6 +470,12 @@ def add_testcase(self, testcase):
         self.append(testcase)
         self.update_statistics()
 
+    def add_testcases(self, testcases):
+        """Adds a testcases to the suite."""
```
[the remark this answers is not in the document]
Comment (Goblenus) [reply junitparser/junitparser.py]: done

File: junitparser/junitparser.py (hunk 1 above)
Comment (Goblenus) [reply junitparser/junitparser.py]: done

**Comment by Goblenus** [comment]
@EnricoMi check please

**Review by EnricoMi — COMMENTED** [review]
+1, LGTM!

**Comment by EnricoMi** [comment]
@weiwei what do you think?

**Comment by weiwei** [comment]
I think this is quite useful. Thanks @Goblenus and @EnricoMi !

**Pull request merged**